README.rst

```diff
@@ -597,5 +597,3 @@ ansible-lint was created by `Will Thames`_ and is now maintained as part of the
 .. _Will Thames: https://github.com/willthames
 .. _Ansible: https://ansible.com
 .. _Red Hat: https://redhat.com
-.. _command-line skip_list: https://docs.ansible.com/ansible-lint/usage/usage.html#command-line-options
-.. _config file skip_list: https://docs.ansible.com/ansible-lint/configuring/configuring.html#configuration-file
```

Comment:
out of scope

Reply:
not really, I was forced to touch these two links because it was failing to build with them. I am not yet sure what confused sphinx about them. Anyway, the second issue of links to wrong docs location is also sorted. Yep, i can split it, but it needs to be addressed before we merge this.